v8_c_bridge.cc

```diff
@@ -25,21 +25,22 @@
 extern "C" ValueErrorPair go_callback_handler(
     String id, CallerInfo info, int argc, PersistentValuePtr* argv);
 
-class ArrayBufferAllocator : public v8::ArrayBuffer::Allocator {
- public:
-  virtual void* Allocate(size_t length);
-  virtual void* AllocateUninitialized(size_t length);
-  virtual void Free(void* data, size_t);
-};
-void* ArrayBufferAllocator::Allocate(size_t length) {
-  void* data = AllocateUninitialized(length);
-  return data == nullptr ? data : memset(data, 0, length);
-}
-void* ArrayBufferAllocator::AllocateUninitialized(size_t length) { return malloc(length); }
-void ArrayBufferAllocator::Free(void* data, size_t) { free(data); }
-
-// We only need one, it's stateless.
-ArrayBufferAllocator allocator;
+// TODO: Figure out why this is causing a linker error.
```

Comment:
yeah, I can't figure out the linker error either, but they did change the ArrayBuffer::Allocator API in 6.0 according to https://docs.google.com/document/d/1g8JFi8T_oAE_7uAri7Njtig7fKaPDfotU6huOa1alds/edit